amespahdbpythonsuite/transitions.py

```diff
@@ -920,8 +920,13 @@ def plot(self, **keywords) -> None:
     @staticmethod
     def absorption_cross_section(f: np.ndarray) -> np.ndarray:
```

Comment:
It would be could to still have the old functionality available as an option--possibly as a separate function and allowing selecting it using a keyword in the emission models.